terraform/modules/lb/backends/lb_backends.tf

```diff
@@ -9,7 +9,7 @@ locals {
 
 resource "oci_load_balancer_backend_set" "wls_lb_backendset" {
   # If using existing load balancer, use per-created backend set of existing lb
-  count = var.use_existing_lb ? 0 : 1
+  count = var.configure_secure_mode ? 0 : var.use_existing_lb ? 0 : 1
```

Comment:
I think we want to be able to use existing load balancers. I'm fine with approving this as is for the topic branch, but this is conditional on there being a backlog ticket for investigating how to support existing load balancers.